### PR TITLE
Add Keras Website Documentation for Layers II

### PR DESCRIPTION
**Pull request by Quincy2014**
## What changes were proposed in this pull request?

Add keras website documentation for core, convolutional layers.

## How was this patch tested?

Jenkins.

### CODE REVIEW

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 1]
```diff
@@ -882,3 +882,356 @@ Output is
...
+
+ [[[0.09347565 0.4754528  0.63618458]
+   [0.08016674 0.21696158 0.83892852]]
+
+  [[0.81864575 0.90813398 0.08347963]
+   [0.57234761 0.76060611 0.65707858]]]]
+```
+Output is
+```python
+[[[[0.0        0.0        0.0        0.0        0.0       ]
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: remove the blank spaces?

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 2]
```diff
@@ -656,3 +656,146 @@ Output is
...
+Masking(maskValue = 0.0, inputShape = null)
+```
+**Python:**
+```python
+Masking(mask_value=0.0, input_shape=None)
+```
+
+**Parameters:**
+
+* `maskValue`: Double, mask value. For each timestep in the input tensor (dimension #1 in the tensor),
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: refactor

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 3]
```diff
@@ -656,3 +656,146 @@ Output is
...
+               if all values in the input tensor at that timestep are equal to `mask_value`, then the timestep will masked (skipped) in all downstream layers.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: new line??

File: docs/docs/KerasStyleAPIGuide/Layers/core.md [hunk 4]
```diff
@@ -656,3 +656,146 @@ Output is
   [0.68505836 1.7576246  0.71985936 1.8025242 ]
   [1.5343305  1.3676738  0.50516313 1.1503248 ]]]
 ```
+
+---
+## **Masking**
+Use a mask value to skip timesteps for a sequence.
+
+Masks a sequence by using a mask value to skip timesteps.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/core.md]: Duplicate description?

**Comment by hkvision** [comment]
I forgot to mention that we need to add `name=None` to ALL layers in Python.

**Comment by hkvision** [comment]
Change `Wrapper Layers` to `Layer Wrappers`

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 5]
```diff
@@ -882,3 +882,653 @@ Output is
...
+  [0.50690836 0.38107997 0.05881034 0.96402144]
+  [0.42226282 0.7735051  0.54961294 0.5531527 ]
+  [0.42226282 0.7735051  0.54961294 0.5531527 ]]]
+```
+
+---
+## **UpSampling2D**
+UpSampling layer for 2D inputs.
+
+Repeats the rows and columns of the data by size(0) and size(1) respectively.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: change to `by the specified size`

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 6]
```diff
@@ -882,3 +882,653 @@ Output is
...
+   [0.55307275 0.55307275 0.33007517 0.33007517]
+   [0.31527168 0.31527168 0.8778964  0.8778964 ]
+   [0.31527168 0.31527168 0.8778964  0.8778964 ]]]]
+```
+
+---
+## **UpSampling3D**
+UpSampling layer for 3D inputs.
+
+Repeats the 1st, 2nd and 3rd dimensions of the data by size(0), size(1) and size(2) respectively.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: by the specified size

**Comment by Quincy2014** [comment]
Jenkins passed. http://172.168.2.101:8080/job/BigDL-PR-Validation-v3/438/

**Pull request merged**